api/src/main/scala/za/co/absa/loginsvc/rest/controller/TokenController.scala

```diff
@@ -145,11 +145,36 @@ class TokenController @Autowired()(jwtService: JWTService) {
   @ResponseStatus(HttpStatus.OK)
   def getPublicKey(): CompletableFuture[PublicKey] = {
     val publicKey = jwtService.publicKey
-    val publicKeyBase64 = Base64.getEncoder.encodeToString(publicKey.getEncoded)
-
+    val publicKeyBase64 = Base64.getEncoder.encodeToString(publicKey._1.getEncoded)
     Future.successful(PublicKey(publicKeyBase64))
   }
 
+  @Tags(Array(new Tag(name = "token")))
+  @Operation(
+    summary = "Gives payload with the current and previously rotated RSA256 public key",
+    description = """Alternative to /public-key - exposes current and previous public keys allowing users to verify a JWT after rotation.""",
+    responses = Array(
+      new ApiResponse(responseCode = "200", description = "Payload containing current and previous public keys is returned",
+        content = Array(new Content(
+          schema = new Schema(implementation = classOf[PublicKey]),
+          examples = Array(new ExampleObject(value = "{\n  \"key\": \"ABCDEFGH1234\"\n}")))
```

Comment:
```suggestion
          examples = Array(new ExampleObject(value = "{\n  \"key\": \"ABCDEFGH1234\"\n  \"previousKey\": \"ZYXWVUT9876\"\n}"))))
```

I think there is still the old description there